ts/src/config.ts

```diff
@@ -78,6 +78,12 @@ export interface Config extends GoogleAuthOptions {
   // stack depth may increase overhead of profiling.
   heapMaxStackDepth?: number;
 
+  // Samples with stacks containing this substring will not be included in
+  // heap profiles.
+  // By default this is set to "@google-cloud/profiler" to exclude samples from
+  // the profiler.
+  ignoreHeapSamplesPath?: string;
```

Comment:
Do we really need this to be configurable? Is this to be able to turn this off during debugging?

Reply:
Yes, this is to be able to turn this off for debugging. 
Should I switch it to a boolean? It might make initializing this string trickier, but it can be done.